CONTRIBUTING.md

```diff
@@ -37,3 +37,27 @@ For example, for running tests in `multi`, `uuid` and `main` packages, call
 ```bash
 make test-multi test-uuid test-main
 ```
+
+## Code review checklist
+
+- Changelog entry is present in `CHANGELOG.md`.
+- There are no changes in files not related to the issue (except for a simple
+  refactoring that obviously brings benefits and does not complicate the
+  review).
+- There are no obvious flaky tests.
+- New features have functional and probably performance tests.
+- Bugfixes have tests based on reproducers.
+- Public functions, variables and constants contains at least a single-line
+  comment.
```

Comment:
Shouldn't we say something regarding being pessimistic in making functions/constants public and start them from a lower case letter if doubt?

Maybe add a suggestion to check how generated documentatoin look (with a short recipe)?

Reply:
Updated.